test/test_sam_annotator/test_annotator_2d.py

```diff
@@ -0,0 +1,54 @@
+import platform
```

Comment:
Should there be a `test_image_series_annotator.py` file to accompany the others in this folder?

Reply:
Good point, I will see if I can add it.